UPCarouselFlowLayout/UPCarouselFlowLayout.swift

```diff
@@ -27,6 +27,7 @@ open class UPCarouselFlowLayout: UICollectionViewFlowLayout {
     
     @IBInspectable open var sideItemScale: CGFloat = 0.6
     @IBInspectable open var sideItemAlpha: CGFloat = 0.6
+    @IBInspectable open var sideItemShift: CGFloat = 70
```

Comment:
As she shift isn't a part of the default expected behavior, the default value should be `0.0`